src/main/java/org/ecommerce/message/broker/consumers/Consumer.java

```diff
@@ -0,0 +1,37 @@
+package org.ecommerce.message.broker.consumers;
+
+import org.ecommerce.logs.Log;
+import org.ecommerce.message.broker.MessageQueue;
+import org.ecommerce.models.Product;
+
+public abstract class Consumer<T> extends Log implements Runnable {
```

Comment:
This class shouldn't extend from `Log`.
If we want to make use of logging features, we can instantiate a `Log` object instead.

Reply:
Ok, that was actually something i was wondering, here is the commit [911f6f6d51f1fe666d518fb616a3f904c7569a77](https://github.com/algutierrez-griddynamics/e-commerce/pull/26/commits/911f6f6d51f1fe666d518fb616a3f904c7569a77)